pytorch/training/docker/1.9/py3/cu111/Dockerfile.gpu

```diff
@@ -0,0 +1,318 @@
+FROM nvidia/cuda:11.1.1-base-ubuntu20.04
+
+LABEL maintainer="Amazon AI"
+LABEL dlc_major_version="1"
+
+ARG PYTHON=python3
+ARG PYTHON_VERSION=3.8.5
+ARG CUBLAS_VERSION=11.3.0.106
+ARG OPEN_MPI_PATH=/opt/amazon/openmpi
+ARG EFA_PATH=/opt/amazon/efa
+
+ARG CUDA_HOME=/usr/local/cuda
+ARG CONDA_PREFIX=/opt/conda
+ARG METIS=metis-5.1.0
+ARG RMM_VERSION=0.15.0
+
+# The smdebug pipeline relies for following format to perform string replace and trigger DLC pipeline for validating
+# the nightly builds. Therefore, while updating the smdebug version, please ensure that the format is not disturbed.
+ARG SMDEBUG_VERSION=1.0.9
```

Comment:
Just double-checking that this is the smdebug version we expect in the final DLC. Could you confirm?

Reply:
@leleamol Could you confirm please?

Reply:
Amol confirmed in another comment, resolving this comment.